### PR TITLE
JWWEB-167 Update DigitalAsset card component

### PR DESCRIPTION
**Pull request by Dragonesis**

### CODE REVIEW

File: src/components/DigitalAssets/Asset/Asset.js [hunk 1]
```diff
@@ -1,103 +1,165 @@
...
+      fiatBalance,
+      isLoading,
+      isLongLoading,
+    } = this.props
+
+    const {
+      isHovered,
+    } = this.state
+
+    const isLoadingContext = isLoading ? 'loading' : isLongLoading ? 'long-loading' : null
```
[the remark this answers is not in the document]
Comment (IViol) [reply src/components/DigitalAssets/Asset/Asset.js]: We shouldn't use nested ternary operators. We need to add eslint rule for this

File: src/components/DigitalAssets/Asset/Asset.js [hunk 2]
```diff
@@ -1,103 +1,165 @@
...
+        onMouseEnter={handle(this.onHover)(true)}
+        onMouseLeave={handle(this.onHover)(false)}
+        className={classNames('asset-card')}
+      >
+        <div className='symbol'>
+          <JAssetSymbol symbol={symbol} color='gray' />
+        </div>
+        <div className='name'>
+          {/*
+            @TODO: consultation with the programmer on the size of the text:
```
Comment (IViol) [inline src/components/DigitalAssets/Asset/Asset.js]: We really mustn't use comments in public repo as your personal notes.

File: src/components/DigitalAssets/Asset/Asset.js [hunk 3]
```diff
@@ -1,103 +1,165 @@
...
+            size 'header' has insufficient size
+          */}
+          <JText value={name} color='dark' weight='bold' size='header' />
+        </div>
+
```
Comment (IViol) [inline src/components/DigitalAssets/Asset/Asset.js]: we don't need to add useless empty lines in jsx. We need to add eslint rule for this

File: src/components/DigitalAssets/Asset/Asset.js [hunk 4]
```diff
@@ -1,103 +1,165 @@
...
+        {(() => {
+          switch (isLoadingContext) {
```
Comment (IViol) [inline src/components/DigitalAssets/Asset/Asset.js]: better to split each case in this switch into separate sub-component

File: src/components/DigitalAssets/Asset/asset.scss [hunk 5]
```diff
@@ -7,73 +7,68 @@
   @include hover-animation;
 
   position: relative;
-  padding: 30px;
-  border: 1px solid transparent;
+  background-color: #fff;
```
Comment (IViol) [inline src/components/DigitalAssets/Asset/asset.scss]: please use scss colors from zepplin styleguide, not hex values

File: src/components/AssetBalance/AssetBalance.js [hunk 6]
```diff
@@ -1,71 +1,48 @@
...
-
 class AssetBalance extends PureComponent<Props> {
   static defaultProps = {
-    symbol: '',
     weight: null,
-    color: 'white',
+    color: 'gray',
     size: 'normal',
-    balance: 0,
-    isTransparent: false,
```
Comment (IViol) [inline src/components/AssetBalance/AssetBalance.js]: boolean props should be always in defaultProps with false value

File: src/components/DigitalAssets/Asset/Asset.js [hunk 7]
```diff
@@ -69,74 +68,16 @@ class AssetCard extends PureComponent<Props, AssetsState> {
...
-                      />
-                    </div>
-                  )}
-                </div>
-              )
-          }
-        })()}
+        <AssetCardBody
+          address={address}
+          fiatMoney={fiatMoney}
```
Comment (IViol) [inline src/components/DigitalAssets/Asset/Asset.js]: `fiatMoney` -> `fiatCurrency`

File: src/components/DigitalAssets/Asset/AssetCardBody/AssetCardBody.js [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+              <AssetBalance
+                color='blue'
+                size='header'
+                symbol={fiatMoney}
+                balance={fiatBalance}
+              />
+            ) : (
+              <div className='message -transparent'>
+                <JText
+                  value='No USD Exchange'
```
Comment (flexxnn) [inline src/components/DigitalAssets/Asset/AssetCardBody/AssetCardBody.js]: USD -> fiatCurrency

**Pull request merged**